modules/@apostrophecms/piece-type/ui/apos/components/AposPiecesManagerToolbar.vue

```diff
@@ -3,6 +3,7 @@
     <template #leftControls>
       <AposButton
         label="Select" :icon-only="true"
+        :disabled="disableSelection"
```

Comment:
It doesn't look like you used this in your implementation. Same for the prop below.

Reply:
Right. A mistake when I merged my work for `AposJoinsManager` and `AposPiecesManager`. Fixed and improved.

Reply:
👍